src/components/header.tsx

```diff
@@ -67,7 +71,18 @@ export const Header: React.FC<Props> = ({ title, ...rest }) => (
     <Title>{title}</Title>
 
     <LanguagePickerContainer>
-      <LanguagePicker />
+      <span
+        css={`
+          font-size: 36px;
+        `}
+      >
+        🇨🇦
```

Comment:
Just to confirm: static canada flag emoji beside language picker?

Reply:
yes!